R/statistic-histogram.R

```diff
@@ -50,13 +50,13 @@
 #' @return Accuracy guarantee for histogram release, given epsilon.
 #' @rdname histogram.getAccuracy
 
-histogram.getAccuracy <- function(mechanism, n.bins, n, epsilon, delta=10^-6, alpha=0.05) {
+histogram.getAccuracy <- function(mechanism, n.bins, n, epsilon, delta=10^-6, alpha=0.05, sensitivity) {
```

Comment:
Do we need to keep `n.bins` and `n` in the signature? It seems these are no longer considered necessary for the accuracy computation. Same arguments for `histogram.getEpsilon` and `n` in `histogram.getCI`.

Reply:
good point